.drone.yml

```diff
@@ -1,3 +1,13 @@
+# ==============================================================================
```

Comment:
Why not use the universal way of using PULL, as here and use DRY (don't repeat yourself!):

 `clone:
  git:
    image: plugins/git:next
+   pull: true`

Reply:
Ah, I didn't know!
Then yes, let's use it

Reply:
I'm wondering if using pull: true in there has the same meaning as putting it in the other service.

Reply:
I have not tried it yet, let's see what drone says!

Reply:
Fixed.